docker-compose.yml

```diff
@@ -9,6 +9,8 @@ services:
       # - /mnt/:/mnt/
       # - /home/pipeline_data/:/tmp/
       - .:/data/pipeline/
+      - ../commons:/data/commons
+      - ~/Documents/Pfaffinger_Reimer/pipeline-externals:/mnt/dj-stor01/pipeline-externals
```

Comment:
do not mount an absolute path like this as that'd make this not usable by anyone else (unless they happen to have ~/Documents/Pfaffinger_Reimer). Rather, keep this a relative path, or draw this information from an environment variable.

Reply:
Good point. This will be a lab share eventually. It'll be good to configure in the `.env` file. Is that possible to set environment variable based on system in the `.env` file? e.g. for mac, /Volumes/dj-stor01, for linux /mnt/dj-stor01

Reply:
ah that's slightly more tricky but we could implement that check on the Python side